<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Account</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Poor Richard</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="password_entry"/>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QListView" name="accounts_list_view"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="search_entry">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Search...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="account_delete_button">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="reset_score_button">
       <property name="text">
        <string>Reset Score</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
